saviSimulation/savi/jason_processing/SAVIWorld_model.java

```diff
@@ -155,26 +155,39 @@ public void setup() {
 		// ======= Jason BDI agents ================
 		Map<String, AgentModel> agentList = new HashMap<String, AgentModel>();
 		//====================================================
-				
+
+
+//		createUGV(agentList, 300, 300);
+		createUAV(agentList, 500, 500);
+		createTree(250, 300);
+		createTree(300, 250);
+		createTree(250, 250);
+		createTree(250, 350);
+//		createTree(350, 250);
+//		createTree(350, 300);
+//		createTree(300, 350);
+//		createTree(350, 350);
+//
+
 		for(int i = 0; i < NUMBER_UAV; i++)  { //Put UaV
 			//_PIXELS is the maximum and the 1 is our minimum
 			//TODO: right now agents are initialized with strings "0", "1", "2", ... as identifiers and a fixed type "demo" which matches their asl file name. This should be configurable...
 			if(RANDOM_SEED != -1) {
 				rand = new Random(RANDOM_SEED+i);
-			}	
```

Comment:
please move the rand initialisation outside the loop. Re-initializing it repeatedly probably doesn't do anything good for the randomness.

Reply:
That is a bug I introduced. Thanks for noticing.
I will open an issue and check all the random numbers. Many will disappear with the map configuration
I think it is better to address this on a different PR.